common/lib/xmodule/xmodule/partitions/partitions.py

```diff
@@ -40,7 +40,7 @@ class Group(namedtuple("Group", "id name")):
 
     def __new__(cls, id, name):
         # pylint: disable=super-on-old-class
-        return super(Group, cls).__new__(cls, int(id), name)
+        return super(Group, cls).__new__(cls, id, name)
```

Comment:
@wedaly sounds good. You are right that we should use int id's for backward compatibility.